### ISSUE TITLE
No translations

### ISSUE THREAD

**Issue by nepozs** [Villhellm/lovelace-clock-card#14]
It's not strictly an issue, but it would be nice if there was a mechanism for use localizations.

**Comment by Villhellm** [comment]
Localizations in what way? The name of the time zone in other languages?

**Comment by nepozs** [comment]
No, I suppose timezone name is standarized, but I mean translation day and month names.
![Screenshot_2020-10-13 Drugi - Home Assistant_edit](https://user-images.githubusercontent.com/56661567/95863924-0b74e400-0d65-11eb-8632-fedbead113c0.png)
so tables
```
    var dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
```
```
    var monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```
could be in separate file 

and for example in polish it should look like
```
    var dayNames = ["Niedz.", "Pon.", "Wt.", "Śr.", "Czw.", "Pt.", "Sob.", "Niedziela", "Poniedziałek", "Wtorek", "Środa", "Czwartek", "Piątek", "Sobota"];
```
```
    var monthNames = ["Sty.", "Lut.", "Mar.", "Kwi.", "Maj", "Cze.", "Lip.", "Sie.", "Wrz.", "Paź.", "Lis.", "Gru.", "Styczeń", "Luty", "Marzec", "Kwiecień", "Maj", "Czerwiec", "Lipiec", "Sierpień", "Wrzesień", "Październik", "Listopad", "Grudzień"];
```


And if this can help I've found some resorces:
official calendar card uses translations and internal HA names for weekdays and months
![calendar Screenshot_2020-10-13 Przegląd - Home Assistant](https://user-images.githubusercontent.com/56661567/95871053-8215df80-0d6d-11eb-847a-5642a4bd136a.png)

and this custom card uses translations (but I don't have any compatible eqipment with this integration…):
https://github.com/denysdovhan/vacuum-card/
I'm not programmer, but I think it has nearly ready-made localization mechanism.

Translated version looks like this (I've done changes directly in .js)
![PL_Screenshot_2020-10-13 Przegląd - Home Assistant](https://user-images.githubusercontent.com/56661567/95899526-ff9f1700-0d90-11eb-8b73-d1935bfc2cf8.png)
or
![PL2_Screenshot_2020-10-13 Lustro - Home Assistant](https://user-images.githubusercontent.com/56661567/95900583-653fd300-0d92-11eb-8f43-914e7aa6c0dc.png)

**Comment by breti** [comment]
Indeed. Localized day and month names would be great.

**Comment by nepozs** [comment]
Similiar project uses localized texts, so maybe this can help?
https://github.com/tomasrudh/analogclock

**Comment by Obliviant** [comment]
that project doesn't show the text without the clock.. My clock has a size of 1px and display's the text as I like that better.. using 2 different cards, one for the date and one for the time.

![Schermopname (9)](https://user-images.githubusercontent.com/1726955/104954182-0fe61400-59c8-11eb-8307-f51015dc2ac3.png)

the other project uses a very simple localisation method.. it basicely uses the HASS language for localisation
```
      var locale = hass.language;
      if (config.locale) locale = config.locale;

```
maybe this helps? would be great! 
or maybe there is a better way to achieve what I did here.. I just started with HA so I have no clue ;-)

**Comment by nepozs** [comment]
@Obliviant 
This project is closed forever
https://www.home-assistant.io/blog/2021/01/06/release-20211/#in-memoriam-of-villhellm

Everyone, who is reading this can help Villhellm's family
https://gofund.me/939d8778

The only way to continue project is fork and active developing…

**Comment by breti** [comment]
I've forked Villhellm's project and added locale support: https://github.com/breti/lovelace-intl-clock-card
